tests/README.md

```diff
@@ -119,17 +119,17 @@ Properties:
 
 Local Test Process:
 ====
-* -Plocal enables failsafe-maven-plugin processing of @Local annotations
+* by default failsafe-maven-plugin processes @LocalOnly annotations
 * com.spotify:docker-maven-plugin builds target container based on value of *jetty.test.image*
 * io.fabric8:docker-maven-plugin starts the target container in pre-integration-test phase
   * random local port mapped to 8080 of container and available to test case as system property *app.deploy.port*
 * failsafe-maven-plugin runs in integration-test phase
-* io.fabric8:docker-maven-plugin stops the target container in  post-integration-test phase
+* io.fabric8:docker-maven-plugin stops the target container in post-integration-test phase
 
 Remote Test Process:
 ====
 
-* -Premote enables failsafe-maven-plugin processing of @Remote annotations
+* -Dtest.remote enables failsafe-maven-plugin processing of @RemoteOnly annotations
```

Comment:
Local is enabled by default using the directive in the activation, and it has test.mode=local property activation as well.  If -Dtest.mode=remote is used that turns off the local profile and the remote ones are activated.  All tests are effectively turned off with -P-test.local in the end. There are different ways to do it, this just seems to work and is fairly straight forward.